File: TechnologiesFundamentals/SoftwareTechnologies/JavaBasics/JavaSyntaxBasicWebCalculator/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {
    private double leftOperand;
    private  double rightOperand;
    private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public double calculateResult() {
        double result;

        switch (this.getOperator()) {
            case "+":
                result = this.getLeftOperand() + this.getRightOperand();
                break;

            case "-":
                result = this.getLeftOperand() - this.getRightOperand();
                break;

            case "*":
                result = this.getLeftOperand() * this.getRightOperand();
                break;

            case "/":
                result = this.getLeftOperand() / this.getRightOperand();
                break;

            default:
                result = 0;
                break;
        }

        return result;
    }
}

File: TechnologiesFundamentals/SoftwareTechnologies/C#/CSharp-ASP.NET-MVC-Overview-Calculator/Calculator-CSharp/Models/Calculator.cs
﻿namespace Calculator_CSharp.Models
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Web;

    public class Calculator
    {
        public Calculator()
        {
            this.Result = 0;
        }

        public decimal LeftOperand { get; set; }

        public decimal RightOperand { get; set; }

        public string Operator { get; set; }

        public decimal Result { get; set; }
    }
}